app/view/static/js/vendor/jquery-throttle-debounce-1.1.js

```diff
@@ -0,0 +1,252 @@
+/*!
+ * jQuery throttle / debounce - v1.1 - 3/7/2010
+ * http://benalman.com/projects/jquery-throttle-debounce-plugin/
+ *
+ * Copyright (c) 2010 "Cowboy" Ben Alman
+ * Dual licensed under the MIT and GPL licenses.
+ * http://benalman.com/about/license/
+ */
```

Comment:
The "debounce" function allows us to only call a function once after a certain period of time. So, if you're typing a query, we don't throw a request on every keypress, but only when you've stopped typing for 150ms (for example).